lib/tapioca/compilers/dsl/active_support_current_attributes.rb

```diff
@@ -113,12 +113,9 @@ def instance_methods_for(constant)
 
         sig { params(klass: Parlour::RbiGenerator::Namespace, method: String, class_method: T::Boolean).void }
         def generate_method(klass, method, class_method:)
-          if method.end_with?("=")
-            parameter = Parlour::RbiGenerator::Parameter.new("value", type: "T.untyped")
-            klass.create_method(method, class_method: class_method, parameters: [parameter], return_type: "T.untyped")
-          else
-            klass.create_method(method, class_method: class_method, return_type: "T.untyped")
-          end
+          parameters = []
```

Comment:
It's a matter of preference but the previous form with a `if .. else` was stating the intent better as the condition was showed upfront. Now the condition is hidden at the end of an already long line.